engine/runtime-instrument-common/src/main/scala/org/enso/compiler/context/ChangesetBuilder.scala

```diff
@@ -31,13 +31,15 @@ import scala.util.Using
   * @param ir the IR node of the module
   * @param simpleUpdate description of a simple editing change (usually in a literal)
   * @param invalidated the list of invalidated expressions
+  * @param idMap the external identifiers
   * @tparam A the source type
   */
 case class Changeset[A](
   source: A,
   ir: IR,
   simpleUpdate: Option[SimpleUpdate],
-  invalidated: Set[UUID @ExternalID]
+  invalidated: Set[UUID @ExternalID],
```

Comment:
Here we use `@ExternalID` - why it is not used everywhere in the new code, @hubertp?